packages/unity-bootstrap-theme/stories/organisms/global-footer/global-footer.templates.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import innovationLockup from "./200420-GlobalFooter-No1InnovationLockup.png";
+import innovationLockup from "./2024-ranked-1.png";
```

Comment:
change
`import innovationLockup from "./2024-ranked-1.png";`
to
`import innovationLockup from "./../../../src/img/innovation-lockup/on-gold/2024-ranked-1.png";`
and delete
`packages/unity-bootstrap-theme/stories/organisms/global-footer/2024-ranked-1.png`

Even better if we can remove the duplication of `innovationLogo.png` and `2024-ranked-1.png` but I don't think we should complicate this ticket since those are in different packages. 

Approved with or without suggested change